en/admin/config.rst

```diff
@@ -1589,6 +1589,14 @@ The following are parameters related to SQL statements and data types supported
         ==========
                0.5
 
+    .. note:: 
+
+        If oracle_compat_number_behavior is set to yes, the data must be read in String format to prevent JDBC/CCI from outputting decimal point trailing zeros for NUMERIC, DOUBLE, and FLOAT types. Please refer to the usage instructions below.
+
+        * In JDBC, you must use the getString(int columnIndex) or getString(String columnLabel) and getObject(int columnIndex) or getObject(String columnLabel) methods.
```

Comment:
* In JDBC, use the getString(int columnIndex) or getString(String columnLabel) and getObject(int columnIndex) or getObject(String columnLabel) methods.

Reply:
I applied mhoh3963's suggestion.